packages/sw-cli/src/cli/index.js

```diff
@@ -370,14 +388,164 @@ class SWCli {
     });
   }
 
-  /**
-   * This method will generate the file manifest only.
-   * @return {Promise} The promise returned here will be used to exit the
-   * node process cleanly or not.
-   */
-  buildFileManifest() {
-    // TODO: Build File Manifest
-    return Promise.resolve();
+  _saveConfigFile() {
+    return inquirer.prompt([
+      {
+        name: 'saveConfig',
+        message: 'Last Question - Would you like to save these settings to ' +
+          'a config file?',
+        type: 'confirm',
+        default: true,
+      },
+    ])
+    .then((results) => {
```

Comment:
Again, slightly fancier, to take advantage of the implicit single-line arrow function return:

```js
.then((results) => results.saveConfig)
```